### ISSUE TITLE
update out-of-scope content on homepage

### ISSUE THREAD

**Issue by ajo2995** [plantbreeding/brapi-website#12]
The standard can't enforce performance claims, so maybe replace/remove "optimized for speed".

Also, the data transfer use case is not there yet (until v2 POST/PUT specs are defined)

**Comment by jackeblagare** [comment]
I agree with this. The standard may be designed in such a way that it promotes good performance but some implementation can mess it up and make even a simple call perform badly. Promoting "ease of data transfer" would be more appropriate I guess?

My understanding with the data transfer use case is that someone can build a client that connects to Server A, extracts data, and save it in their own database. Using that same client, it can then connect to Server B and do the same thing.

**Comment by BrapiCoordinatorSelby** [comment]
I reworded the "Optimized for Speed" section to the following
>A lot of effort has been invested to make BrAPI an efficient data model without compromising flexibility. Implementations can be tuned to fit the scale of the database.

Let me know what you think, or if it still needs to just be taken out. 

And V2 is out now, so data transfer makes sense now :)

Sorry this took a year to fix
